### PR TITLE
Don't override $(PATH) environment variable.

### PR DESCRIPTION
**Pull request by shishir-a412ed**

### CODE REVIEW

**Comment by shishir-a412ed** [comment]
@chuckyz Can you please review?

Tested this with our `redis-containerd` job in `alpha` and `$PATH` is not getting overridden.

```
(alpha) SM-smahajan628:example smahajan$ nomad alloc exec -i -t de1515d3 env
PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
REDIS_VERSION=6.0.8
REDIS_DOWNLOAD_URL=http://download.redis.io/releases/redis-6.0.8.tar.gz
REDIS_DOWNLOAD_SHA=04fa1fddc39bd1aecb6739dd5dd73858a3515b427acd1e2947a66dadce868d68
```

File: containerd/driver.go [hunk 1]
```diff
@@ -358,6 +358,10 @@ func (d *Driver) StartTask(cfg *drivers.TaskConfig) (*drivers.TaskHandle, *drive
 	// Setup environment variables.
 	var env []string
 	for key, val := range cfg.Env {
+		// Don't override $PATH.
```
Comment (chuckyz) [inline containerd/driver.go:361]: Let's setup a future proof thing here for any keys that need to be other than PATH and have it default to just PATH

Comment (shishir-a412ed) [reply containerd/driver.go:361]: https://play.golang.org/p/v5gGhQQ-9_7

Comment (shishir-a412ed) [reply containerd/driver.go]: Fixed.

**Review by chuckyz — APPROVED** [review]
LGTM

**Pull request merged**